common/pb/response.proto

```diff
@@ -37,6 +37,7 @@ message Response {
         RespRegistrationAcceptedNeedsActivation = 33; // Server accepted cient registration, but it will need token activation
         RespClientIdRequired = 34; // Server requires client to generate and send its client id before allowing access
         RespClientUpdateRequired = 35; // Client is missing features that the server is requiring
+        RespUserLimitReached = 36; // Server has a user limit and it has been reached
```

Comment:
Nope, in the past the lookup was done in the session creation step and thus an event was generated. The session creation happens before the actual user information checks occur and a response to login is not generated but an event is returned.  Having written the original code for this check and coming back to it later I see how implementing it in the portion of the code that does the login code portion makes more since than in the session creation portion.